aipo/README.md

```diff
@@ -0,0 +1,4 @@
+# AipO
+Administration and International Business Pensum's Ontology from Peruvian Union University 
+
+More info at https://w3id.org/aipo/aipo
```

Comment:
I don't think that will match any rules, did you mean to just use https://w3id.org/aipo?